apps/site/pages/en/about/branding.mdx

```diff
@@ -17,6 +17,8 @@ Please review the [trademark policy](https://trademark-policy.openjsf.org/) for
   height="114"
 />
 
+Credit to [@Ang_ngl on X](https://x.com/Ang_ngl) for designing and contributing the Rocket Turtle.
```

Comment:
IMHO, we should put this paragraph before the image. And perhaps mention that the mascot was chosen in a vote?

Reply:
Great point. If it were just a simple credit, the reason I like having it after is it follows what I'm familiar with in media/literature of having the credit _after_ the work. But if adding a bit of context around the origin of the mascot, then that does make more sense before the image. Could go either way IMO, maybe someone else senior would have an opinion?

Reply:
Hello! I am new to open source and I was wondering how you found out who the credit should go to? I want to understand how to navigate through codespaces and resolve issues.

Reply:
> Hello! I am new to open source and I was wondering how you found out who the credit should go to? I want to understand how to navigate through codespaces and resolve issues.

Hello! Honestly I was wondering the same thing so I googled the mascot and found the twitter post from the NodeJS account crediting the artist. I just copied the link

Reply:
Oh I see, thank you!

Reply:
I saw it as a context, so it's smarter to put it on top. But if it's presented as a biobliographical reference, then yes, it should be put underneath.

Reply:
> I saw it as a context, so it's smarter to put it on top. But if it's presented as a biobliographical reference, then yes, it should be put underneath.

Perhaps reference for now, and then if the team wants to add more information like the vote etc it could be a separate issue? The vote and deciding process is unfamiliar to me so I doubt I could do the event justice in the description. Also, did I miss anything on translation or is it good to go @AugustinMauroy ?

Reply:
Actually I forget you brought up the issue, it's your call. I have to fix my GPG commit signing so I'll switch it to above and commit shortly.